pwnlib/ui.py

```diff
@@ -21,6 +19,8 @@
 log = getLogger(__name__)
 
 def testpwnproc(cmd):
+    import fcntl
+    import termios
```

Comment:
This is good, will also possibly speed up the whole thing, since these imports are just for the tests.